packages/components/MIGRATING.md

```diff
@@ -162,6 +162,108 @@ return (
 </Flyout.Root>
 ```
 
+### Dialog
+
+Changes:
+
+-   The `Dialog` component now controls its open state internally by default.
+
+    -   You can pass in a trigger component as a child of `Dialog.Trigger` to control the open state.
+
+    -   The `open` and `onOpenChange` props can still be used to control the open state externally if needed.
+
+-   The `Dialog` now provides multiple subcomponents.
+
+    -   _required_ - The `Dialog.Root` wraps all other subcomponents and handles the flyout state and modality.
```

Comment:
Please at least read the sentences you are copy/pasting...
Do you document flyout or dialog here?

Reply:
i made some misxups when rebasing

Reply:
fixed now